app/models/feed.rb

```diff
@@ -10,9 +10,7 @@ def get(limit, max_id = nil, since_id = nil)
     max_id     = '+inf' if max_id.blank?
     since_id   = '-inf' if since_id.blank?
     unhydrated = redis.zrevrangebyscore(key, "(#{max_id}", "(#{since_id}", limit: [0, limit], with_scores: true).map(&:last).map(&:to_i)
-    status_map = Status.where(id: unhydrated).cache_ids.map { |s| [s.id, s] }.to_h
-
-    unhydrated.map { |id| status_map[id] }.compact
+    Status.where(id: unhydrated).cache_ids
```

Comment:
It looks like previously part of what was being collected in the map was a `[s.id, s]` pair, which presumably is a status id and a full status record.

Does the change here to just return the `cache_ids` result (which I think is going to be an array of [:id, :updated_at]` for each status?) change anything w/ how this is ultimately used?

I don't know enough about how `cache_ids` and `cache_collection` and redis are intended to interact here to have a solid opinion.